forms-flow-api/src/formsflow_api/services/employeeDataService.py

```diff
@@ -54,9 +58,8 @@ def get_employee_names(args):
                        headers={"Authorization": current_app.config.get("ODS_AUTH_TOKEN")})
 
       except Exception as e:
-        raise BusinessException(
-          {"message": "Failed to get employee names from ODS"}, HTTPStatus.INTERNAL_SERVER_ERROR
-        )
+        current_app.logger.error(e)
```

Comment:
I have omitted the error raise statement.
any error is stop search functionality in the Form "remote select" component. 

I have logged an error here, So if we receive any query from QA or User we can take look and see what new case we have to handle here to search from the ODS.